src/main/resources/org/jenkinsci/plugins/githubautostatus/BuildStatusConfig/help-statsdBucket.html

```diff
@@ -0,0 +1,14 @@
+<div>
+    Bucket to be prepended to the metrics you are sending (with trailing period). Example:
+    <ul>
+        <li>service.jenkins.pipeline-metrics.</li>
```

Comment:
The other statsd metrics are being sent as `service.jenkins-kubernetes.pipeline.` should they math or do they need to be different?